lib/token_util.rb

```diff
@@ -23,4 +23,11 @@ def self.to_asset_group_name(wildcard)
     return wildcard if wildcard.nil?
     wildcard.gsub('?', '')
   end
+
+  def self.generate_positions(letters, columns)
+    size=letters.size * columns.size
+    location_for_position = size.times.map do |i|
+      "#{letters[(i/columns.length).floor]}#{(columns[i%columns.length]).to_s}"
```

Comment:
I think the location string syntax should have padded 2-digit numbers, for example "A01" rather than "A1", to be consistent with code elsewhere.